src/main/java/uk/gov/pay/ledger/transaction/state/RefundState.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.pay.ledger.transaction.state;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.Arrays.stream;
+import static java.util.List.of;
+import static uk.gov.pay.ledger.transaction.state.TransactionState.CREATED;
+import static uk.gov.pay.ledger.transaction.state.TransactionState.ERROR_GATEWAY;
+
+public enum RefundState {
+
+    SUBMITTED(of(TransactionState.SUBMITTED, CREATED), "Refund submitted"),
+    SUCCESS(of(TransactionState.SUCCESS), "Refund success"),
+    ERROR(of(TransactionState.ERROR, ERROR_GATEWAY), "Refund error");
```

Comment:
here too, although I quite like the fact the first enum tells you the type of the rest of the list...

Reply:
same here too